.github/workflows/template-e2e-test/action.yaml

```diff
@@ -1,23 +1,25 @@
-# Template for e2e tests.
+# Composite action for e2e tests.
+name: Run E2E Test
+description: Run e2e test using the minikube cluster
 
 inputs:
   experiments:
     required: true
-    type: string
+    description: comma delimited experiment name
   training-operator:
     required: false
-    type: boolean
-    default: false
+    description: whether to deploy training-operator or not
+    default: "false"
```

Comment:
Needs to specify string data.

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault